app/scenes/Document/components/Document.tsx

```diff
@@ -540,6 +542,7 @@ class DocumentScene extends React.Component<Props> {
                         <Contents
                           headings={this.headings}
                           isFullWidth={document.fullWidth}
+                          onLeftSide={tocOnLeftSide}
```

Comment:
```suggestion
                          position={position}
```

Lets continue to pass `position` down